doc/classes/TreeItem.xml

```diff
@@ -253,7 +253,7 @@
 		<method name="get_next" qualifiers="const">
 			<return type="TreeItem" />
 			<description>
-				Returns the next sibling TreeItem in the tree or a null object if there is none.
+				Returns the next sibling TreeItem in the tree or a [code]null[/code] object if there is none.
```

Comment:
While there is a distinction between `null` and `object null`, nowhere else in the documentation have I ever seen it. It usually does not matter to the end user, and it's rather verbose. I would rather just say `null`.